nsis/pcsx2_full_install.nsi

```diff
@@ -45,6 +53,69 @@ ShowUninstDetails nevershow
 ; =======================================================================
 
 ; -----------------------------------------------------------------------
+
+; The "" makes the section hidden.
+Section "" SecUninstallPrevious
+
+	Call UninstallPrevious
+	
+SectionEnd
+
+Function UninstallPrevious
+	;Check for uninstaller.
+	ReadRegStr $R0 HKLM "${INSTDIR_REG_KEY}" "UninstallString"
+	${If} $R0 == ""
+		Goto Done
+	${EndIf}
+	
+	;Check if any other version is installed
+	ReadRegStr $R1 HKLM "${INSTDIR_REG_KEY}" "InstalledVersion"
+	
+	;This check for older versions (pre 1.6.0) without InstalledVersion string will bypass this section
+	${If} $R1 == ""
+	DetailPrint "InstalledVersion string not found, skipping version check"
```

Comment:
@danilaml I knew I had a good reason for including this check. 

To answer your question, we can't detect the uninstaller executable on older versions is because the uninstaller filename is currently named like so: ```Uninst-pcsx2 1.4.0.exe```

The ```${APP_VERSION}``` string had to be removed to allow for the uninstaller script to function because we can't predict what version the user will have installed. Detecting those older versions wouldn't work unless we defined each registry string individually - which isn't worth the extra installer code.